nautobot/extras/forms/forms.py

```diff
@@ -1774,6 +1775,39 @@ class ObjectChangeFilterForm(BootstrapMixin, forms.Form):
 #
 
 
+class RelationshipBulkEditForm(BootstrapMixin, CustomFieldModelBulkEditFormMixin, NoteModelBulkEditFormMixin):
+    pk = forms.ModelMultipleChoiceField(queryset=Relationship.objects.all(), widget=forms.MultipleHiddenInput())
+    label = forms.CharField(max_length=CHARFIELD_MAX_LENGTH, required=False)
+    description = forms.CharField(max_length=CHARFIELD_MAX_LENGTH, required=False)
+    type = forms.ChoiceField(choices=RelationshipTypeChoices, required=False)
+    source_hidden = forms.NullBooleanField(required=False, widget=BulkEditNullBooleanSelect)
+    destination_hidden = forms.NullBooleanField(required=False, widget=BulkEditNullBooleanSelect)
+    key = forms.CharField(max_length=CHARFIELD_MAX_LENGTH, required=False)
+    source_filter = forms.JSONField(required=False, widget=forms.Textarea, help_text="Filter for the source")
+    destination_filter = forms.JSONField(required=False, widget=forms.Textarea, help_text="Filter for the destination")
+    source_type = forms.ModelChoiceField(queryset=ContentType.objects.all(), required=False)
+    destination_type = forms.ModelChoiceField(queryset=ContentType.objects.all(), required=False)
```

Comment:
These should maybe use `CSVContentTypeField`, and should definitely add a FeatureQuery("relationships") so that irrelevant content-types aren't included.